### DISCUSSION TITLE
Support custom CA certs

### DISCUSSION THREAD

**Discussion by lapsio** [GitSquared/edex-ui#998] [Ideas]
App doesn't properly use system-wide installed CA certificates thus fails when used behind SSL Proxy with following log:

```
✖  error     Error: self signed certificate in certificate chain: Uncaught Error: self signed certificate in certificate chain
⬤  debug     at events.js 287:7
✖  error     Error: self signed certificate in certificate chain: Uncaught Error: self signed certificate in certificate chain
⬤  debug     at events.js 287:7
✖  error     Error: self signed certificate in certificate chain: Uncaught Error: self signed certificate in certificate chain
⬤  debug     at events.js 287:7
●  note      UpdateChecker: Could not fetch latest release from GitHub's API.
⬤  debug     Error: Error: self signed certificate in certificate chain
```

I have installed proxy CA cert system-wide and NodeJS installed locally doesn't have any issues with https connections. However edex-ui doesn't seem to respect system CA. Is there any workaround for this issue? (build in use - 2.2.4)

**Comment by GitSquared** [comment]
Unfortunately eDEX does not support system proxies or custom certificate chains. I've moved your question to an Idea so we can track whether we want to add that feature and when.
